automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/analysis/WorkspaceAnalysisPage.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.firecloud.page.workspaces.analysis
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.{FireCloudConfig, FireCloudView}
+import org.broadinstitute.dsde.firecloud.component.Component._
+import org.broadinstitute.dsde.firecloud.component._
+import org.broadinstitute.dsde.firecloud.page.PageUtil
+import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspacePage
+import org.broadinstitute.dsde.workbench.model.google.GcsPath
+import org.broadinstitute.dsde.workbench.service.test.WebBrowserUtil
+import org.openqa.selenium.WebDriver
+import org.scalatest.selenium.{Page, WebBrowser}
+
+class WorkspaceAnalysisPage(namespace: String, name: String)(implicit webDriver: WebDriver)
+  extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceAnalysisPage] with LazyLogging {
+
+  override def awaitReady(): Unit = {
+    await condition igvContainerSelector.findElement.exists(_.isDisplayed)
```

Comment:
this might be problematic because `.isDisplayed` is not a reliable method to use to determine page finished rendering. I don't know this page well. Is there any additional page objects that should be checked in `awaitReady`? It's okay if there isn't anything else. we can wait and find out in time.

Reply:
What specifically is problematic about `.isDisplayed`? Are there specific use cases that cause problems? The only other element we could test for is the "Select Tracks..." button.